docs/shared/tutorials/gradle.md

```diff
@@ -109,8 +109,7 @@ even intelligently run a subset of tasks based on the changes you've made in you
 computation caching to reuse the results of tasks. We will explore how Nx adds to the task running Gradle provides.
 
 Before we start running tasks, let's explore the tasks available for the `application` project. The `@nx/gradle` plugin
-that we've added, also reflects Gradle's tasks to Nx, allowing it to run any of the Gradle
-tasks. You can do this either through [Nx Console](/getting-started/editor-setup) or from the terminal:
+that we've installed allows Nx to run any of the Gradle tasks defined for that project. You can do this either through [Nx Console](/getting-started/editor-setup) or from the terminal:
```

Comment:
Can you please bring back the word "reflects"?

1. It provides more detail about how exactly Nx is able to run the Gradle tasks.
2. It describes exactly what we're doing
3. It's a concept that Java developers will be familiar with https://www.oracle.com/technical-resources/articles/java/javareflection.html

Reply:
I added it back, but I still think it is weird to call Nx reading Gradle's tasks the same thing as code reflection.  I think it creates more confusion than it clarifies.